RayLin.md

```diff
@@ -1,16 +1,13 @@
-# 03/07 ~ 03/18
-
-- [Study Group]
-  - Firefox Architecture & Product sharing
+# 03/21 ~ 03/25
 
 - [Firefox for Android]
-  - Bug 1065076 - Update icons for video controls
-    - RESOLVED FIXED
-    - No friendly development tool for Android front end, rebuild and reinstall are needed for every change.
   - Bug 1250741 - Update video player spec
     - WIP
-  - Environment setup
-    - Restore Z3C to stock ROM
-    - git-cinnabar
-    - Andriod Studio integration
-
+    - feedback? Margaret
+- [Firefox for Desktop]
```

Comment:
I was planning to take desktop bug like https://bugzilla.mozilla.org/show_bug.cgi?id=1257599 last week, but has been taken by someone. Now I'm assessing the scope of https://bugzilla.mozilla.org/show_bug.cgi?id=1229727 to see what I can do it. If you have another bug recommended for me, I'm very glad to take it. Thanks!